docs/en/ingest-management/elastic-agent/elastic-agent-providers.asciidoc

```diff
@@ -148,38 +145,37 @@ Provides information about the current host. The available keys are:
 [[env-provider]]
 ==== Env Provider
 
-Provides access to the environment variables as key/values.
+Provides access to the environment variables as key and values.
 
-For example, if you set the variable foo:
+For example, set the variable `foo`:
 
 [source,shell]
 ----
 foo=bar elastic-agent run
 ----
 
-You can reference the environment variable as `${env.foo}`.
+The environment variable can be referenced as `${env.foo}`.
 
 [discrete]
 [[kubernetes_secrets-provider]]
 ==== Kubernetes Secrets Provider
 
 Provides access to the Kubernetes Secrets API.
 
-Provider needs a `kubeconfig` file so as to establish connection to Kubernetes API,
-or it can automatically reach the API if it runs in an inCluster environment (Agent runs as Pod).
+The provider needs a `kubeconfig` file to establish connection to the Kubernetes API.
+It can automatically reach the API if it's run in an inCluster environment (agent runs as Pod).
 
 [source,yaml]
 ----
 providers.kubernetes_secrets:
   #kube_config: /Users/elastic-agent/.kube/config
 ----
 
-You can reference the Kubernetes Secrets variable as `${kubernetes_secrets.default.somesecret.value}`,
+Reference the Kubernetes Secrets variable as `${kubernetes_secrets.default.somesecret.value}`,
```

Comment:
Should secret be capitalized? I didn't think so but I wanted to ask before I changed it.

Reply:
Looking at the Kubernetes documentation, yes, I think it should: https://kubernetes.io/docs/concepts/configuration/secret/